ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1738,6 +1738,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-ingress-glb
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le-cucushift
+  cron: 20 23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ppc64le and @ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes
+        and not @fips'
+      E2E_SKIP_TAGS: not @upgrade and not @customer and not @destructive and not @flaky
+        and not @inactive and not @long-duration and not @prod-only and not @qeci
+        and not @security and not @stage-only and not @upgrade-check and not @upgrade-prepare
+      GO_VERSION: 1.20.7
+      JQ_VERSION: "1.6"
+      LOKI_ENABLED: "false"
+      OCP_VERSION: "4.15"
+      RESOURCE_GROUP: ipi-cicd-resource-group
+      SIZE_VARIANT: large
+      TERRAFORM_VERSION: 1.5.5
+      WORKERS: "2"
+      YQ_VERSION: 4.30.5
```

Comment:
IMO, GO_VERSION/JQ_VERSION/OCP_VERSION/TERRAFORM_VERSION/YQ_VERSION are not needed here.  @jianlinliu could you help check ?

Reply:
These are passed into the workflow for Multi-Architecture Compute ... so we can setup the workflow for Intel/Power

Reply:
Sounds they are used by `upi-install-heterogeneous-powervs`, I have no experience on that step. Just have a glance on the step, there are several curl command to download something, I think that need some enhancement, because they probably hit github bottleneck limit when multiple jobs run on the same time.

Reply:
that's right.

Reply:
It's suggested to run those curl/download in a Dockerfile to make up a image, then run your step in a pod with the image, which should be faster and stabler, as curl failures in step will abort the CI, while the failures in Dockerfile is that the image is not refreshed, but it does not stop your CI tests.

Not a blocker to this PR.

Reply:
We will work on it. Thanks @liangxia